models/weatherSchema.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const weatherSchema = mongoose.Schema({
+  minTem : String,
+  maxTem : String, 
```

Comment:
I thought the weather data include strings like ℃ or m/s or etc.
I checked it doesn't.